app/diagnostics.go

```diff
@@ -166,9 +166,7 @@ func (a *App) trackActivity() {
 		inactiveUserCount = iucr.Data.(int64)
 	}
 
-	if tcr := <-a.Srv.Store.Team().AnalyticsTeamCount(); tcr.Err == nil {
-		teamCount = tcr.Data.(int64)
-	}
+	teamCount, _ = a.Srv.Store.Team().AnalyticsTeamCount()
```

Comment:
The point of ignoring the error here is to not fail the whole thing because of this, but I agree that we should do something with the error, my suggestion would be just log the error.